Server/src/main/java/net/simon987/server/assembly/instruction/SetccInstruction.java

```diff
@@ -0,0 +1,327 @@
+package net.simon987.server.assembly.instruction;
+
+import net.simon987.server.assembly.Instruction;
+import net.simon987.server.assembly.Status;
+import net.simon987.server.assembly.Target;
+import net.simon987.server.assembly.Operand;
+import net.simon987.server.assembly.OperandType;
+import net.simon987.server.assembly.MachineCode;
+
+import net.simon987.server.assembly.exception.AssemblyException;
+import net.simon987.server.assembly.exception.IllegalOperandException;
+import net.simon987.server.assembly.exception.InvalidMnemonicException;
+import net.simon987.server.logging.LogManager;
+
+import java.io.ByteArrayOutputStream;
+
+import java.util.Map;
+import java.util.HashMap;
+
+/**
+ * Implementation of the SETcc family of instructions
+ * http://www.ousob.com/ng/iapx86/ng22d84.php
+ * 
+ * 
+ * Stuff to consider
+ * - move the implementation of each instruction to its own class that extends this class
```

Comment:
Since this is (in the eyes of the assembler/cpu) a single instruction with many aliases, it wouldn't make sense to move the implementation to sub classes anyway